config/locales/en.yml

```diff
@@ -108,6 +109,7 @@ en:
       preferred_compensation:
         help_text: What is your preferred annual salary and/or hourly rate?
         title: Preferred compensation
+      processing: Processing
```

Comment:
I'm not a huge fan of processing, it's a tad too technical, in my opinion. I'd love for it to say `Creating...` or `Updating...` depending on if we are creating or updating the record. If that's not possible or not worth the effort, `Submitting...` or `Saving...` could work!

Reply:
The first option sounds way better, so I'll go for that